src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  background-color: #f4f4f4;
+  font-family: Roboto, sans-serif;
+
+  --accent-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo-link {
+  display: flex;
+}
+
+.nav-links {
+  list-style: none;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. Consider adding a class to the `<ul>` element in your HTML and style it using that class.